content/300-guides/200-deployment/201-serverless/500-deploy-to-netlify.mdx

```diff
@@ -287,58 +118,22 @@ Because Prisma will connect to the database from your serverless functions, you
    ![Netlify environment variables](images/500-07-deploy-to-netlify-environment-variables-settings.png)
 5. Click **Save**.
 
-### Result
-
-The `DATABASE_URL` environment variable is now set in your Netlify build environment.
-
-However, the previous deployment was not built with the variable in place and it cannot use that.
+</details>
 
-### What's next
-
-Start a new Netlify build and deployment so that the new build can use the new `DATABASE_URL` environment variable.
+Now start a new Netlify build and deployment so that the new build can use the newly uploaded environment variables.
 
 ```terminal no-copy
 netlify deploy
 ```
 
-## 9. Test the deployed app
-
 You can now test the deployed application.
 
-### Steps
-
-Open the site with the Netlify CLI.
-
-```terminal
-netlify open:site
-```
-
-### Result
-
-Your default browser opens the static frontend that you deployed.
-The URL should look like this: `https://YOUR_SITE_NAME.netlify.app`.
-
-![Netlify application with Prisma deployed](images/500-08-deploy-to-netlify-application-deployed.png)
-
-The four buttons allow you to make requests to the REST API and view the response:
-
-- **Check API status**. Calls the REST API status endpoint that returns `{"up":true}`. The implementation code is in `functions/index.js`
-- **Seed data**. Deletes all database records and loads the database with test data `users`, `profiles`, and `posts`. Returns the created users. The implementation code is in `functions/seed.js`
-- **Load users with profiles**: Loads all `users` in the database with their related `profiles`. The implementation code is in `functions/getUsers.js`
-- **Load Posts**: Loads `posts` and their related `authors`. The implementation code is in `functions/getPosts.js`
-
-For example, when you click **Seed data**, the application shows the following:
-
-![Netlify application with Prisma deployed - Call result](images/500-09-deploy-to-netlify-application-deployed-call-result.png)
-
 ## Summary
 
 You have successfully configured continuous deployments and integration with Netlify for an application that uses Prisma.
 
 Every time you push changes to the `main` branch of the repository, Netlify starts a new production build and deployment. If you push changes to another branch, Netlify starts preview builds and deployments.
 
-For more insight into Prisma Client API, you can explore the function handlers in the `functions/` folder.
-
 When you use a FaaS (function as a service) environment to interact with a database, it is beneficial to pool database connections for performance reasons. This is because every function invocation may result in a new connection to the database (this is not a problem with a constantly running node.js server).
 
 You can use the [Data Proxy](/data-platform/data-proxy) for connection pooling, to reduce your Prisma Client bundle size, and to avoid cold starts.
```

Comment:
Super weird paragraphs. Doesn't really fit the "summary" headline. We should probably change how we include these in the future.

Reply:
Yeah this Conclusion/Summary paragraph is in a number of our existing guides and needs to be re-thought.